cmd/bridge-device/bridge-device.jsonld

```diff
@@ -0,0 +1,117 @@
+{
+    "@context": "https://www.w3.org/2019/wot/td/v1",
+    "@type": [
+        "Thing"
+    ],
+    "id": "urn:uuid:bridge",
+    "properties": {
+        "/oic/d": {
+            "title": "Device Information",
+            "type": "object",
+            "properties": {
+                "piid": {
+                    "title": "Protocol Interface ID",
+                    "type": "string",
+                    "readOnly": true,
+                    "format": "uuid"
+                },
+                "my-property": {
```

Comment:
my property?

Reply:
I see it in the main of the bridge-device, so I think it is expected and not a forgotten debug